backup.py

```diff
@@ -31,128 +32,137 @@
     print("Backup canceled: Library import failed.")
     exit()
 
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 # creates a file called 'backup.log'
 backupLog = open(logLoc, 'a')
-backupLog.write("\n" + datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup script ran.\n")
+backupLog.write(
+    "\n" + datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup script ran.\n")
 
 
 # checks if the config file exists. if it doesn't, it will attempt to retrieve it from github
 
 if not os.path.exists("backupcfg.py"):
-    cfgUrl = "https://raw.githubusercontent.com/30023344/Backup-Script/d55509ceb6656491c48592b0df6ece17aff9d1f5/backupcfg.py" # doesnt work, link is not permanent 
+    # doesnt work, link is not permanent
+    cfgUrl = "https://raw.githubusercontent.com/30023344/Backup-Script/d55509ceb6656491c48592b0df6ece17aff9d1f5/backupcfg.py"
     try:
         import urllib.request
         urllib.request.urlretrieve(cfgUrl, "backupcfg.py")
-        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "'backupcfg.py' was created. Retrieved from '%s'.\n"%cfgUrl)
+        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') +
+                        "'backupcfg.py' was created. Retrieved from '%s'.\n" % cfgUrl)
         print("'backupcfg.py' was created.")
     except:
-        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup Canceled: couldn't retrieve 'backupcfg.py' from '%s'.\n"%cfgUrl)
-        print("Backup canceled: Couldn't retrieve 'backupcfg.py' from '%s'."%cfgUrl)
+        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') +
+                        "Backup Canceled: couldn't retrieve 'backupcfg.py' from '%s'.\n" % cfgUrl)
+        print("Backup canceled: Couldn't retrieve 'backupcfg.py' from '%s'." % cfgUrl)
         exit()
 
-from backupcfg import *
-
 
 # function for writing messages in log, where it also writes the time
 # the first argument given when you call the function tells it what to write after the time stamp
 # the second argument given tells it if you want it to print to console or not
 
 def writeToLog(message, write):
-    backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + message + "\n")
-    if write == True and writeToConsole == True: # will only print a message if both the writeToConsole anhd write variables are set to True
+    backupLog.write(datetime.now().strftime(
+        '%Y-%m-%d %H:%M:%S ') + message + "\n")
+    # will only print a message if both the writeToConsole anhd write variables are set to True
+    if write == True and WRITE_TO_CONSOLE == True:
         print(message)
 
-# this bit copys files
+# this bit copies files
+
+
 def copyFiles():
     try:
         if os.path.isdir(n) == False:
             shutil.copyfile(n, target)
-            writeToLog(pathlib.PurePath(n).name + " was copied to %s."%backupDir, True)
+            writeToLog(pathlib.PurePath(n).name +
```

Comment:
```suggestion
            writeToLog(pathlib.PurePath(jobPath).name +
```
'n' got changed to 'jobPath' but wasn't changed here. I like that the names were changed to be more readable, I was too lazy to do this myself.

Reply:
Sorry my dude, I got sloppy :/